core/sequences/sequences-docs.factor

```diff
@@ -678,7 +678,7 @@ HELP: replace-slice
 { $description "Replaces a range of elements beginning at index " { $snippet "from" } " and ending before index " { $snippet "to" } " with a new sequence." }
 { $errors "Throws an error if " { $snippet "new" } " contains elements whose types are not permissible in " { $snippet "seq" } "." } ;
 
-{ push prefix suffix } related-words
+{ push push-all push-either push-if pop pop* prefix suffix suffix! } related-words
```

Comment:
That sounds good. I don't really have a strong opinion just think we should have a general guidance that's all.